subprojects/testing-base/src/main/java/org/gradle/api/tasks/testing/AbstractTestTask.java

```diff
@@ -566,6 +575,22 @@ void setFailFast(boolean failFast) {
         this.failFast = failFast;
     }
 
+    /**
+     * Indicates if this task will succeed when there is no test to run.
+     *
+     * @return whether this task will succeed when there is no test to run
+     */
+    Property<Boolean> getSuccessWithoutTest() {
```

Comment:
:x: This needs to be marked as `@Internal` otherwise other descendent of `AbstractTestTask` than `Test` will fail task properties validation